doc/user_manual/sampler.tex

```diff
@@ -105,6 +105,14 @@ \section{Samplers}
   \end{itemize}
 }
 
+\newcommand{\constantVariablesDescription}
+{
+  \xmlNode{constant}, \xmlDesc{XML node,
+  optional parameter} the user is able to input variable that needs to be kept constant.    
+  For doing this, as many \xmlNode{constant} nodes as needed can be inputted, where the body of the node containing the constant value
```

Comment:
"inputted" isn't an (English) word, I believe you can just use "input", or maybe it would be more clear as "specified" or "defined" or "added".  "Input" is probably just as good.